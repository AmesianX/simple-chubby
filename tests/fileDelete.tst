echo Test:fileDelete starts.
# create, open again, then close both
fileOpen /test 8
fileOpen /test 3
fileClose 1
fileClose 2
# open and delete
fileOpen /test 3
fileDelete 3
fileClose 3
# create and delete
fileOpen /test 8
fileDelete 4
fileClose 4

echo Test finished.
exit

