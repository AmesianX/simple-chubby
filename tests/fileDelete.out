Test:fileDelete starts. 
1
2
3
Deleted
4
Deleted
Test finished. 
